packager/mpd/base/adaptation_set.h

```diff
@@ -269,6 +269,9 @@ class AdaptationSet {
   // Determined by examining the MediaInfo passed to AddRepresentation().
   std::string content_type_;
 
+  // Codec of AdaptationSet
+  std::string codec_;
```

Comment:
Also define an accessor for codec_ after line 178:

/// @return codec.
std::string codec() const { return codec_; }